src/transaction.js

```diff
@@ -222,4 +226,57 @@ export class Transaction extends TransactionBase {
 
     return envelope;
   }
+
+  /**
+   * Calculate the claimable balance ID for an operation within the transaction.
+   *
+   * This helper throws under the following circumstances:
+   *    - invalid `opIndex` value
+   *    - the operation at the `opIndex` is not a CreateClaimableBalance op
+   *    - the source account doesn't have a sequence number
+   *    - general XDR un/marshalling failures
```

Comment:
💡  I _believe_ you could use multiple `@throws` tags here, something like:
```js
/*
 * (...)
 * @throws {RangeError}  if opIndex is invalid
 * @throws {TypeError}   if the operation at the `opIndex` is not a CreateClaimableBalance op
 * (...)
/*
```

At least that works great in javadoc, so I believe it's the same for jsdoc.

This change is not required BTW, it's just a suggestion.

Reply:
Great call, I love it!